depends/packages/qt.mk

```diff
@@ -180,6 +180,9 @@ $(package)_config_opts_mingw32 += "QMAKE_LFLAGS = '$($(package)_ldflags)'"
 $(package)_config_opts_mingw32 += "QMAKE_LIB = '$($(package)_ar) rc'"
 $(package)_config_opts_mingw32 += -device-option CROSS_COMPILE="$(host)-"
 $(package)_config_opts_mingw32 += -pch
+ifneq ($(LTO),)
+$(package)_config_opts_mingw32 += -ltcg
+endif
```

Comment:
FWIW, adding the `-ltcg` option saves 512 bytes only of `bitcoin-qt.exe` size being built in Guix.

Reply:
bc1qdrc8fup30ctv7ppjyr2txcg6wgcrejj2nk53we